<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogEvent</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resurce.qrc">
    <normaloff>:/img/log_viewer.ico</normaloff>:/img/log_viewer.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="btnConnectDb">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resurce.qrc">
              <normaloff>:/img/extDatabase.png</normaloff>:/img/extDatabase.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnCloseDb">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resurce.qrc">
              <normaloff>:/img/db_close.png</normaloff>:/img/db_close.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>5</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolBtnUpdate">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resurce.qrc">
              <normaloff>:/img/tbglacti_100.png</normaloff>:/img/tbglacti_100.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="maximumSize">
             <size>
              <width>5</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnOpenFilterDlg">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resurce.qrc">
              <normaloff>:/img/filterandsort.png</normaloff>:/img/filterandsort.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnResetFilter">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resurce.qrc">
              <normaloff>:/img/resetFilter.png</normaloff>:/img/resetFilter.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnSetFilterCurrentValue">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resurce.qrc">
              <normaloff>:/img/filterCurrentValue.png</normaloff>:/img/filterCurrentValue.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>5</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chLimit">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Первые</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinLimit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Период:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dtStaretDate">
            <property name="maximumSize">
             <size>
              <width>148</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>по:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dtEndDate">
            <property name="maximumSize">
             <size>
              <width>145</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnSelectInterval">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resurce.qrc">
              <normaloff>:/img/selectInterval.png</normaloff>:/img/selectInterval.png</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="mnuOpenSrvinfo"/>
    <addaction name="separator"/>
    <addaction name="mnuDbConnect"/>
    <addaction name="mnuDbClose"/>
    <addaction name="separator"/>
    <addaction name="mnuExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>&amp;Сервис</string>
    </property>
    <addaction name="mnuColumnVisuble"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>&amp;Помощь</string>
    </property>
    <addaction name="mnuAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mnuExit">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="mnuOptions">
   <property name="text">
    <string>Параметры</string>
   </property>
  </action>
  <action name="mnuDbConnect">
   <property name="icon">
    <iconset resource="resurce.qrc">
     <normaloff>:/img/extDatabase.png</normaloff>:/img/extDatabase.png</iconset>
   </property>
   <property name="text">
    <string>Открыть базу данных</string>
   </property>
  </action>
  <action name="mnuDbClose">
   <property name="icon">
    <iconset resource="resurce.qrc">
     <normaloff>:/img/db_close.png</normaloff>:/img/db_close.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть базу данных</string>
   </property>
  </action>
  <action name="mnuOpenSrvinfo">
   <property name="text">
    <string>Открыть каталог srvinfo</string>
   </property>
  </action>
  <action name="mnuColumnVisuble">
   <property name="text">
    <string>Видимость колонок</string>
   </property>
  </action>
  <action name="mnuAbout">
   <property name="text">
    <string>О программе ...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resurce.qrc"/>
 </resources>
 <connections/>
</ui>
